sdk/src/types/block/slot/commitment.rs

```diff
@@ -19,7 +19,7 @@ use crate::types::block::{
     serde(rename_all = "camelCase")
 )]
 pub struct SlotCommitment {
-    // The version of the protocol running.
+    /// The version of the protocol running.
```

Comment:
weird comment

Reply:
yup, just made it a proper doc comment. I don't mind changing this doc in this PR. Suggestions?

Reply:
`The protocol version` is fine tbh